packages/ui/src/features/core/editable-table/editable-table-collapsed-row.tsx

```diff
@@ -18,10 +19,14 @@ export const EditableTableCollapsedRow = ({
   colSpan,
   rowId,
   tableRow,
+  disabled = false,
   setExpanded,
   onDelete,
 }: EditableTableCollapsedRowProps) => {
   const { classes } = useStyles()
+  const disabledStyle = {
+    color: 'rgba(0,0,0,0.2)',
```

Comment:
Should this style be fetched from theme/useStyles?

Reply:
Moved to style file